<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_name">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="tab_key">
     <attribute name="title">
      <string>Keys</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>791</width>
        <height>521</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_keys">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QListView" name="listView_keys"/>
       </item>
       <item>
        <widget class="QFrame" name="frame_keys">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_ship">
     <attribute name="title">
      <string>Ships</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>791</width>
        <height>521</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_ship">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QListView" name="listView_ship"/>
       </item>
       <item>
        <widget class="QFrame" name="frame_ship">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QComboBox" name="comboBox_ship_categories">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>391</width>
            <height>28</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_fleet">
     <attribute name="title">
      <string>Fleets</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>791</width>
        <height>521</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_fleet">
       <item>
        <widget class="QListView" name="listView_fleet"/>
       </item>
       <item>
        <widget class="QFrame" name="frame_fleet">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QComboBox" name="comboBox_fleet_name_categories">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>391</width>
            <height>28</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_army">
     <attribute name="title">
      <string>Armies</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>791</width>
        <height>521</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_army">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QListView" name="listView_army"/>
       </item>
       <item>
        <widget class="QFrame" name="frame_army">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QComboBox" name="comboBox_army_name_categories">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>391</width>
            <height>28</height>
           </rect>
          </property>
         </widget>
         <widget class="QComboBox" name="comboBox_army_name_type">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>40</y>
            <width>391</width>
            <height>28</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_planet">
     <attribute name="title">
      <string>Planets</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>791</width>
        <height>521</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_planet">
       <item>
        <widget class="QListView" name="listView_planet"/>
       </item>
       <item>
        <widget class="QFrame" name="frame_planet">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QComboBox" name="comboBox_planet_categories">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>391</width>
            <height>28</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_character">
     <attribute name="title">
      <string>Characters</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>791</width>
        <height>521</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_character">
       <item>
        <widget class="QListView" name="listView_character"/>
       </item>
       <item>
        <widget class="QFrame" name="frame_character">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QComboBox" name="comboBox_character_name_set">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>391</width>
            <height>28</height>
           </rect>
          </property>
         </widget>
         <widget class="QComboBox" name="comboBox_character_name_categories">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>40</y>
            <width>391</width>
            <height>28</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_character_name_set_weight">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>90</y>
            <width>113</width>
            <height>28</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Name_List"/>
    <addaction name="actionOpen_Localization"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Name_List">
   <property name="text">
    <string>Open Name List</string>
   </property>
  </action>
  <action name="actionOpen_Localization">
   <property name="text">
    <string>Open Localization</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
